frontend/src/components/Credentials/GCredentialRowInfra.vue

```diff
@@ -44,7 +44,22 @@ SPDX-License-Identifier: Apache-2.0
       </v-tooltip>
     </td>
     <td v-if="selectedHeaders.credential">
-      <span v-if="!item.hasOwnCredential">{{ item.credentialNamespace }}: </span>{{ item.credentialName }}
+      <span v-if="item.isSharedCredential">{{ item.credentialNamespace }}: </span>{{ item.credentialName }}
```

Comment:
typically this is written as namespace/name

Reply:
replaced with /